src/lib/utils/scales-utils.js

```diff
@@ -293,7 +293,8 @@ function _computeRightDomainAdjustment(values) {
  */
 function _computeScaleDistance(values, domain, index, scaleFn) {
   if (values.length > 1) {
-    return Math.abs(scaleFn(values[index]) - scaleFn(values[index - 1]));
+    const i = index > 0 ? index : 1;
```

Comment:
Math.max would also do? Otherwise, I think this is clearer if you did `index >= 1`